specs/consensus.md

```diff
@@ -176,30 +179,51 @@ For `wrappedTransaction`'s [transaction](./data_structures.md#transaction) `tran
 
 1. `transaction.signature` must be a [valid signature](./data_structures.md#public-key-cryptography) over `transaction.signedTransactionData`.
 
-Finally, each `wrappedTransaction` is processed depending on [its transaction type](./data_structures.md#signedtransactiondata). These are specified in the next subsections, where `tx` is short for `transaction.signedTransactionData`, and `sender` is the recovered signing [address](./data_structures.md#address). After applying a transaction, the new state state root is computed.
+Finally, each `wrappedTransaction` is processed depending on [its transaction type](./data_structures.md#signedtransactiondata). These are specified in the next subsections, where `tx` is short for `transaction.signedTransactionData`, and `sender` is the recovered signing [address](./data_structures.md#address). We will define a helper functions:
+```
+baseCost(y) = block.header.feeHeader.baseRate * y
+tipCost(y) = block.header.feeHeader.tipRate * y
+totalCost(x, y) = x + baseCost(y) + tipCost(y)
```

Comment:
Would it make sense to state the domain of these functions here?

Reply:
Sure.

Reply:
Fixed in 4a951c6.